### DISCUSSION TITLE
Discord Server for the mod?

### DISCUSSION THREAD

**Discussion by OliverMatt2023** [Sinytra/Connector#50] [General]
You should make a discord server for this mod. Many developers have one. If you want, I can make one for you if you'd like. A few people were wanting a discord for this mod. 

let's say it gives more options for feedback and more interaction between the dev and the community.

**Comment by hexadecimal233** [comment]
@Su5eD Could you create a discord server when you are free? We realllly want it.

**Comment by Su5eD** [comment]
I agree having a discord server is an important part of interacting with the community these days, and we will definitely get one too. However, it's not as simple as just creating a server. The moderation tools, roles, channels etc all need to be set up. So it might take some time to fully prepare it.

**Comment by Su5eD** [comment]
The discord server is ready. You can join using this link: https://discord.gg/mamk7z3TKZ
